src/utils/nations.js

```diff
@@ -45,4 +49,20 @@ export function nationIsValid(nation) {
   if (_.isEmpty(nation.governanceService)) return false;
 
   return true;
-}
\ No newline at end of file
+}
+
+// @todo Use Pangea libs implementation.
+export function resolveListStyle(nation): nationStatus {
+  if (nation.created) {
```

Comment:
Please do not copy the implementation of resolveStatus, but reuse it instead. So, if it's changed eventually, we don't need to update both methods.

Reply:
Done some refactor